test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_mnist_inductor.py

```diff
@@ -94,8 +94,13 @@ def test_mnist_distributed_graviton_inductor(
 def test_mnist_distributed_gpu_inductor(
     framework_version, ecr_image, instance_type, sagemaker_regions
 ):
-    if Version(framework_version) in SpecifierSet("<2.0"):
-        pytest.skip("skip the test as torch.compile only supported after 2.0")
+    if Version(framework_version) in SpecifierSet("<2.0") or Version(
+        framework_version
+    ) in SpecifierSet("==2.2.0"):
+        pytest.skip(
+            f"skip the test as torch.compile only supported after 2.0"
```

Comment:
is it "after **and including 2.0**"?

Reply:
No, only 2.2.0. fails. 2.0 and 2.1 still works. This is just a single instance of 2.2.0 binary failing. We'll release 2.2.0 as planned first because the fix is in 2.2.1.

Reply:
Got it!

nit: Since `2.0` still works, maybe we can change the skip message to say that its supported for `>=2.0`.